File: app/src/main/java/me/cargoapp/cargo/event/navui/HandleNavuiActionAction.java
package me.cargoapp.cargo.event.navui;

/**
 * Created by Marvin on 26/04/2017.
 */

public class HandleNavuiActionAction {
    Type _type;

    public HandleNavuiActionAction(Type type) {
        _type = type;
    }

    public Type getType() {
        return _type;
    }

    public enum Type {
        CALL,
        MESSAGE,
        MUSIC,
        OIL,
        PARKING,
        MENU,
        QUIT
    }
}

File: app/src/main/java/me/cargoapp/cargo/event/navui/HandleNavuiActionAction.kt
package me.cargoapp.cargo.event.navui

/**
 * Created by Marvin on 26/04/2017.
 */

data class HandleNavuiActionAction(val type: Type) {
    enum class Type {
        CALL,
        MESSAGE,
        MUSIC,
        OIL,
        PARKING,
        MENU,
        QUIT
    }
}
